src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -5661,6 +5665,1805 @@ int P2
         }
 
         [Fact]
+        public void MethodModifiers_01()
+        {
+            var source1 =
+@"
+public interface I1
+{
+    public void M01();
+    protected void M02();
+    protected internal void M03();
+    internal void M04();
+    private void M05();
+    static void M06();
+    virtual void M07();
+    sealed void M08();
+    override void M09();
+    abstract void M10();
+    extern void M11();
+    async void M12();
+}
```

Comment:
Feature "private protected" will like go into master branch in 7.1. Maybe take a note to add a test for it when it does and you merge latest bits from master into your feature branch.

Is "new" be in the list of modifiers that you're testing?